pocs/utils/logger.py

```diff
@@ -72,9 +199,9 @@ def get_root_logger(profile='panoptes', log_config=None):
         # missing from the log_config. It is hard to understand how
         # this could occur given that none of the callers of
         # get_root_logger pass in their own log_config.
-        if 'formatters' not in log_config and sys.stdout.isatty():
-            import pdb
-            pdb.set_trace()
+        if 'formatters' not in log_config:  # pragma: no cover
+            warn('formatters is missing from log_config!')
```

Comment:
So since  #672 we can use things from `pocs.utils.error` within the other util classes. This might be a good place to raise a custom exception rather than the `warn`.

Reply:
Added a TODO.